### PR TITLE
DeltaV species guidebook entries

### PR DESCRIPTION
**Pull request by Axionyxx**
## About the PR
I copied these straight from the wiki.

I noticed that we didn't have any while testing loadouts

**Changelog**
:cl:
- add: Vulpkanin, felinids, oni and harpies now have guidebook entries

### CODE REVIEW

File: Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ## Diet
+  - Nothing special.
+
+  ## Benefits
+  - Takes 15% less Blunt, Slash and Piercing damage.
+  - Does more damage in melee; 35% more Blunt and Asphyxiation, 20% more Slash and Piercing.
+  - Due to their big size, they can easily carry, shove and pull any other morphotypes.
+  - They are harder to shove.
+  - Slightly more stamina than Humans.
+  - They have an easier type prying open doors, taking around 50% less time than other species.
```
Comment (deltanedas) [inline Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml]: time

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml (hunk 1 above)
Comment (Axionyxx) [reply Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml]: Huh? I just copied it from the wiki

Comment (deltanedas) [reply Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml]: easier time not easier type

Comment (Axionyxx) [reply Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml]: Ohhh, I see it now

Comment (Axionyxx) [reply Resources/ServerInfo/Guidebook/Mobs/DeltaV/Oni.xml]: Fixed

**Review by deltanedas — APPROVED** [review]

**Pull request merged**